site/config-overrides.md

```diff
@@ -205,61 +205,61 @@ The following `jdbc-testDS.xml` override template demonstrates setting the URL,
+  * Templates must not override the settings listed in [Unsupported overrides](#unsupported-overrides).
+  * Templates must be formatted and named as per [Override template names and syntax](#override-template-names-and-syntax) and [References](#references).
+  * Templates can embed macros that reference environement variables or Kubernetes secrets.  See [Override template macros](#override-template-macros).
+* Create a Kubernetes configuration map from the directory of templates.
+  * The configuration map must be in the same Kubernetes namespace as the domain.
+  * If the configuration map is going to be used by a single `DOMAIN_UID`, it is recommended to add the `weblogic.domainUID=<mydomainuid>` label to help track the resource.
+  * For example, assuming `./mydir` contains your `version.txt` and situation configuration template files:
     ```
     kubectl -n MYNAMESPACE create cm MYCMNAME --from-file ./mydir
     kubectl -n MYNAMESPACE label cm MYCMNAME weblogic.domainUID=DOMAIN_UID
     ```
-* Create any kubernetes secrets referenced by a template macro.
-  * Secrets can have multiple keys (files) that can hold either cleartext or base64 values
-  * Secrets must be in the same kubernetes namespace as the domain
-  * If a secret is going to be used by a single DOMAIN_UID, it is recommended to add the 'weblogic.domainUID=<mydomainuid>' label to help track the resource.
+* Create any Kubernetes secrets referenced by a template macro.
+  * Secrets can have multiple keys (files) that can hold either cleartext or base64 values.
+  * Secrets must be in the same Kubernetes namespace as the domain.
+  * If a secret is going to be used by a single `DOMAIN_UID`, it is recommended to add the `weblogic.domainUID=<mydomainuid>` label to help track the resource.
   * For example:
     ```
     kubectl -n MYNAMESPACE create secret generic my-secret --from-literal=key1=supersecret --from-literal=key2=topsecret
     kubectl -n MYNAMESPACE label secret my-secret weblogic.domainUID=DOMAIN_UID
     ```
-* Configure the name of the config map in the Domain CR `configOverrides` field.
-* Configure the names of each secret in Domain CR.
-  * If the secret contains the WebLogic admin `username` and `password` keys, set the Domain CR `webLogicCredentialsSecret` field.
-  * For all other secrets, add them to Domain CR `configOverrideSecrets` field.
-* See [Debugging](#debugging) for ways to check if sit cfg is taking effect or if there are errors.
+* Configure the name of the configuration map in the domain CR `configOverrides` field.
+* Configure the names of each secret in domain CR.
+  * If the secret contains the WebLogic admin `username` and `password` keys, set the domain CR `webLogicCredentialsSecret` field.
+  * For all other secrets, add them to domain CR `configOverrideSecrets` field.
+* See [Debugging](#debugging) for ways to check if the situational configuration is taking effect or if there are errors.
 
 ---
 # Debugging
```

Comment:
Debugging should include some info like

The following flags can be added in the domain resource file to enable Debug.

env:
    - name: JAVA_OPTIONS
      value: "-Dweblogic.debug.DebugSituationalConfig=true -Dweblogic.debug.DebugSituationalConfigDumpXml=true"

Reply:
Rosemary, please add a final bullet to the debugging section, that's something like:  To cause the WebLogic Situational Config feature to produce additional debugging information in the WebLogic Server logs, configure the "JAVA_OPTIONS" env var in your domain resource with "-Dweblogic.debug.DebugSituationalConfig=true -Dweblogic.debug.DebugSituationalConfigDumpXml=true".

Reply:
Got it. Done.